server.js

```diff
@@ -1,13 +1,15 @@
+require("dotenv").config(); // this ensures process.env. ... contains your .env file configuration values
+
 const express = require('express');
 const bodyParser = require('body-parser');
 const Web3 = require('web3');
 const config = require('./config.json');
 
-const walletPrivateKey = process.env.walletPrivateKey;
-const web3 = new Web3('https://mainnet.infura.io/v3/_your_api_key_here_');
+const walletPrivateKey = process.env.walletPrivateKey; // make sure this is provided in your .env file
+const web3 = new Web3(`https://mainnet.infura.io/v3/${process.env.infuraProjectId}`);
 
 web3.eth.accounts.wallet.add(walletPrivateKey);
-const myWalletAddress = web3.eth.accounts.wallet[0].address;
```

Comment:
In my opinion deriving the wallet address from providing the private key can lead to some misunderstandings. 

If a user e.g. uses metamask, exports his account's private key and enters it in walletPrivateKey, this would not necessarily reveal the correct wallet address in web3.eth.accounts.wallet[0].address, therefore I'd let the user enter the walletAddress also in the .env file.